src/network/network.cpp

```diff
@@ -839,33 +863,16 @@ void Network::LoadGroups()
 	platform_get_user_directory(path, NULL);
 	strcat(path, "groups.json");
 
-	if (!platform_file_exists(path)) {
+	json_t * json = nullptr;
+	try {
+		json = Json::ReadFromFile(path);
+	} catch (const Exception& e) {
+		log_error("Failed to read %s as JSON. Setting default groups. %s", path, e.what());
```

Comment:
windows.h annoyingly defines a GetMessage macro, I didn't use what() because it wasn't UpperCamelCase, and GetMsg was just a temporary workaround for the GetMessage macro issue.